horovod/runner/elastic/discovery.py

```diff
@@ -15,20 +15,33 @@
 
 import io
 import logging
+import random
 import threading
-
+import time
 from collections import defaultdict
 
 from horovod.runner.common.util import safe_shell_exec
 from horovod.runner.elastic.worker import HostUpdateResult
-
-
```

Comment:
I think we should keep at least one empty line here

Reply:
Done